pkg/idtools/idtools.go

```diff
@@ -277,3 +280,18 @@ func parseSubidFile(path, username string) (ranges, error) {
 	}
 	return rangeList, nil
 }
+
+func checkChownErr(err error, name string, uid, gid int) error {
+	if e, ok := err.(*os.PathError); ok && e.Err == syscall.EINVAL {
+		return errors.Wrapf(err, "there might not be enough IDs available in the namespace (requested %d:%d for %s)", uid, gid, name)
```

Comment:
How about 
"%d:%d dones have have a valid mapping in the user namespace for %s"

Reply:
we are not sure that is the problem without checking `/proc/self/uid_map`, that is why I've left it as a possibility in the message.  Even though I don't see other reasons why it could fail with that error code.

Reply:
I don't know if you want to add something about "verify the values in /proc/self/uid_map" or some such to that.

Reply:
there is need to be in the correct namespace to verify `/proc/self/uid_map`

Reply:
I just want something more specific for the user. then "Their might not be" ...